test/info.test.ts

```diff
@@ -17,16 +17,20 @@ describe("commands/info", function () {
 
   test("throws with invalid table name", async function () {
     const consoleError = spy(console, "error");
-    await yargs(["info", "invalid_name"]).command(mod).parse();
+    await yargs(["info", "invalid_name", "--chain", "local-tableland"])
+      .command(mod)
+      .parse();
     assert.calledWith(
       consoleError,
       "invalid table name (name format is `{prefix}_{chainId}_{tableId}`)"
     );
   });
 
-  test("throws with invalid chain", async function () {
+  test.skip("throws with invalid chain", async function () {
     const consoleError = spy(console, "error");
-    await yargs(["info", "valid_9999_0"]).command(mod).parse();
+    await yargs(["info", "valid_9999_0", "--chain", "local-tableland"])
+      .command(mod)
+      .parse();
```

Comment:
Looks like this is just being skipped. Not sure if it is actually fixed.

Reply:
Oops. I've removed the skip. I've also removed the specifying of the chain, since the command infers that from the tablename.